assets/css/style.css

```diff
@@ -1,138 +1,157 @@
-<<<<<<< HEAD
-
-  @font-face {
-    font-family: poke;
-    src: url("../fonts/pokemon-font.otf");
-  }
-
-  .svg-container {
-    position: absolute;
-    top: 0;
-    right: 0;
-    left: 0;
-    z-index: -1;
-  }
-  
-  body {
-    background: #fff;
-    position: relative;
-    background: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/652/confectionary.png); 
-  }
-
-  header {
-    color: #fff;
-    padding-top: 10vw;
-    text-align: center;
-  } 
-=======
 * {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-  }
-
-  @font-face {
-    font-family: poke;
-    src: url("/assets/fonts/pokemon-font.otf");
-  }
-
-  body {
-    font-family: poke, sans-serif;
-  }
-
-  header {
-    text-align: center;
-    margin-top: 30px;
-    overflow: hidden;
-  }
->>>>>>> 80e81584bc845beebeea7b87fa00fb27182f06e5
-
-  header h1 {
-    text-transform: uppercase;
-  }
-
-<<<<<<< HEAD
-  /* Float five columns side by side */
-  .poke-cards {
-    float: left;
-    width: 20%;
-    padding: 0 20px;
-  }
-
-  /* Remove extra left and right margins, due to padding */
-  .row {margin: 0 150px;}
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: poke;
+  src: url("../fonts/pokemon-font.otf");
+}
+
+body {
+  background: #fff;
+  position: relative;
+  font-family: poke;
+  font-size: large;
+  background: url(https://s3-us-west-2.amazonaws.com/s.cdpn.io/652/confectionary.png);
+}
+
+header {
+  color: #fff;
+  padding-bottom: 1vw;
+  text-align: center;
+  overflow: hidden;
+}
+#logo img {
+  padding-top: 150px;
+}
+
+.svg-container {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: -1;
+}
+
+.user-form {
+  display: flex;
+  flex-direction: row;
+  width: 70%;
+  margin: auto;
+}
+
+.user-form .user-input {
+  flex: 1;
+  display: flex;
+  outline: none;
+}
+
+.user-form .user-input .lab {
+  width: 120px;
+}
+
+.user-form .user-input .form-control {
+  flex: 1;
+  outline: none;
+}
+
+.user-form .user-btn .btn {
+  padding: 8px 10px;
+  outline: none;
+}
+
+/* Display city search */
+.citysearch {
+  text-align: center;
+  margin-bottom: 150px;
+  top: 100px;
+  color: yellow;
+  line-height: 40px;
+}
+
+/* Display weather */
+.weather-container {
+  padding-top: 100px;
+  padding-left: 150px;
+  line-height: 40px;
+  margin-bottom: 100px;
+  margin-top: 50px;
+  color: darkgreen;
+}
 
-  /* Style the counter cards */
-  .card,.card.hn {
-  padding: 20px;
+/* Display submit button */
+.submitsection {
   text-align: center;
-  background-color: #f1f1f1;
-  height:40vh;
-  transform-style: preserve-3d;
+  margin-bottom: 200px;
+  top: 100px;
+  line-height: 100px;
+}
+
+/* Display 5 cards in a row */
+.poke-cards {
+  display: grid;
+  grid-template-columns: repeat(5, 2fr);
+  grid-column-gap: 50px;
+  grid-row-gap: 1em;
+  margin-bottom: 150px;
+  padding: 0 100px;
+}
+
+/* Style the counter cards */
+.card,
+.card.hn {
+  padding: 20px;
+  text-align: left;
```

Comment:
please add: 
text-transform: capitalize; 
- to capitalize the words of the card